static/panes/device-view.ts

```diff
@@ -317,29 +425,130 @@ export class DeviceAsm extends MonacoPane<monaco.editor.IStandaloneCodeEditor, D
         }
     }
 
-    onMouseMove(e: monaco.editor.IEditorMouseEvent): void {
-        if (e.target.position === null) return;
-        if (this.settings.hoverShowSource) {
-            this.clearLinkedLines();
-            const hoverCode = this.deviceCode[e.target.position.lineNumber - 1];
-            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-            if (hoverCode && this.compilerInfo.editorId != null) {
-                // We check that we actually have something to show at this point!
-                const sourceLine = hoverCode.source && !hoverCode.source.file ? hoverCode.source.line : -1;
-                this.eventHub.emit('editorLinkLine', this.compilerInfo.editorId, sourceLine, -1, 0, false);
-                this.eventHub.emit(
-                    'panesLinkLine',
-                    this.compilerInfo.compilerId,
-                    sourceLine,
-                    -1,
-                    0,
-                    false,
-                    this.getPaneName(),
-                );
+    async onMouseMove(e: any) {
+        if (e === null || e.target === null || e.target.position === null) return;
+        // const hoverShowSource = this.settings.hoverShowSource === true;
+        // const hoverAsm = this.assembly[e.target.position.lineNumber - 1];
+        // // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+        // if (hoverShowSource && hoverAsm) {
+        //     this.clearLinkedLines();
+        //     // We check that we actually have something to show at this point!
+        //     let sourceLine = -1;
+        //     let sourceColBegin = -1;
+        //     let sourceColEnd = -1;
+        //     if (hoverAsm.source) {
+        //         sourceLine = hoverAsm.source.line;
+        //         if (hoverAsm.source.column) {
+        //             sourceColBegin = hoverAsm.source.column;
+        //             sourceColEnd = sourceColBegin;
+        //         }
+        //
+        //         const editorId = this.getEditorIdBySourcefile(hoverAsm.source);
+        //         if (editorId) {
+        //             this.eventHub.emit('editorLinkLine', editorId, sourceLine, sourceColBegin, sourceColEnd, false);
+        //
+        //             this.eventHub.emit(
+        //                 'panesLinkLine',
+        //                 this.id,
+        //                 sourceLine,
+        //                 sourceColBegin,
+        //                 sourceColEnd,
+        //                 false,
+        //                 this.getPaneName(),
+        //                 editorId,
+        //             );
+        //         }
+        //     }
+        // }
```

Comment:
When you're ready for the final touches to the PR, please remove the commented out code that's no longer needed